obsproject/obs-plugintemplate
# Add in-tree build support

### Description


Resolve #10 

- The plugin template can be built outside or inside OBS Studio Build tree.
- Linkage to the frontend API and Qt are "disabled" since they are not used by the example.
- I tried to keep the template simple

Note: The Windows only RC file was not added, I just don't know if it's needed for testing and debugging.

### Motivation and Context



I just wanted to do it.

### How Has This Been Tested?



- Build the plugin out of tree, and test the plugin with OBS Studio
- Build the plugin in OBS Studio build tree, and run the built OBS Studio and check if the plugin is built and loaded.

### Types of changes


- New feature (non-breaking change which adds functionality)




- Documentation (a change to documentation pages)

### Checklist:
